### PR TITLE
English Translation for README caver-java-example

### PR DESCRIPTION
**Pull request by jiwon-lieb**
Englisch Translation for README caver-java-example

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,75 @@
-# caver-java-boilerplate
+# caver-java-examples
```
Comment (jiwon-lieb) [inline README.md:4]:
```suggestion
*Read this in Korean:* [Korean](https://github.com/klaytn/caver-java-examples/blob/master/README.ko.md) 


# caver-java-examples
```

**Comment by aeharvlee** [comment]
@minishell 
Hi Elena. Please review this PR :)
Thanks in advance.

File: README.md [hunk 2]
```diff
@@ -1,23 +1,78 @@
...
+    * Make sure to also save the keystore file, because some scenarios require it.
+2. Obtain 5 testnet KLAY on [Baobab Klaytn Wallet Faucet](https://baobab.wallet.klaytn.com/faucet).
+
+### 2. How to Use
+1. Clone this project and then execute `$ git clone https://github.com/klaytn/caver-java-examples.git` in Terminal.
+2. To run the samples, you need to set the Credential (AccessKey ID and Secret AccessKey) which can be obtained on [KAS Console](https://console.klaytnapi.com/ko/security/credential) as well as a Klaytn account obtained on [Klaytn Wallet Testnet](https://baobab.wallet.klaytn.com/create). We recommend using the test account just for testing.
+    * Option 1 (recommended): Define the Credential information and your Klaytn account in the **caver-java-examples/.env** file. The variables defined in this file apply for all scenarios. You can find the code for retrieving the variables defined in the **.env** file in the `loadEnv` method within **src/main/java/CaverExample.java**.
+    * Option 2: Open the **CaverExample.java** file for a specific scenario and fill in the variables declared as `private static`, defined unter the **CaverExample** class.
```
Comment (minishell) [inline README.md]:
```suggestion
    * Option 2: Open the **CaverExample.java** file for a specific scenario and fill in the variables declared as `private static`, defined under the **CaverExample** class.
```

**Review by minishell — COMMENTED** [review]
@terrikwak 리뷰 완료했습니다.

**Comment by jiwon-lieb** [comment]
> @terrikwak 리뷰 완료했습니다.

넵 반영했습니다~!!

File: README.md [hunk 3]
```diff
@@ -1,23 +1,78 @@
...
+### 1.1 Installing Software
+In order to follow this tutorial, you have to first install the software you need for creating this project, as listed below.
+* [OpenJDK 8 (LTS)](https://adoptopenjdk.net/?variant=openjdk8&jvmVariant=hotspot)
+* [Gradle 7.1](https://gradle.org/install/)
+
+### 1.2 Sign Up on KAS
+You can use the Klaytn SDK([caver-java](https://github.com/klaytn/caver-java)) by connecting to one of the nodes that comprise the Klaytn network. But for the caver-js-examples project, we are using the [Node API](https://refs.klaytnapi.com/en/node/latest) of [KAS](https://klaytnapi.com), which means that you don't have to run an [Endpoint Node (EN)](https://docs.klaytn.com/node/endpoint-node) yourself to test the code samples.
+
+To use KAS, first sign up on [KAS](https://klaytnapi.com), and obtain the `AccessKey ID` and `Secret AccessKey`.
+You can find detailed instructions on how to use the `AccessKey ID` and `Secret AccessKey` in [2. How to Use](https://github.com/klaytn/caver-java-examples/blob/master/README.md#2-%EC%82%AC%EC%9A%A9%EB%B2%95).
```
Comment (aeharvlee) [inline README.md]:
```suggestion
You can find detailed instructions on how to use the `AccessKey ID` and `Secret AccessKey` in [2. How to Use](https://github.com/klaytn/caver-java-examples#2-how-to-use).
```

**Review by minishell — APPROVED** [review]

**Pull request merged**